### ISSUE TITLE
feature: Make "Zoom to 100" more apparent in the UI

### ISSUE THREAD

**Issue by LauraWebdev** [uxbox/uxbox#2450]
### Is your feature request related to a problem? Please describe.

I've only known about this shortcut after going through them all in the keymap. You can press Shift + 0 to reset your zoom to 100%. I wasn't aware of this shortcut before because, unlike with the other zoom options, there is no shortcut hint in the UI.

### Describe the solution you'd like.

Display the shortcut Shift + 0 for resetting the zoom in the same area as Zoom to fit and Zoom to selected.

### Describe alternatives you've considered.

This is just a small suggestion, as you can see all shortcuts in the keymap

### Additional context

<img width="234" alt="image" src="https://user-images.githubusercontent.com/7365286/195733824-fb388f4d-2807-4692-81dd-3dccfc6a8678.png">

**Comment by elhombretecla** [comment]
Agree. We will approach this from the design team to find the better way to make it more evidence.

Also, If you want to make a design proposal of this enhancement we are open to any design contribution :)
